libs/ui/src/components/organisms/MemberCard/MemberCard.tsx

```diff
@@ -6,7 +6,7 @@ import {
   truncateAddress,
   ValidNetwork,
 } from '@daohaus/common-utilities';
-import { AccountProfile } from '@daohaus/dao-data';
+import { AccountProfile } from '@daohaus/common-utilities';
```

Comment:
this is the main change of this pr -- moving the type into common utilities removes dao-data from the UI lib